cluster/v1alpha1/types_rolloutstrategy.go

```diff
@@ -64,6 +64,8 @@ type RolloutConfig struct {
 	MinSuccessTime metav1.Duration `json:"minSuccessTime,omitempty"`
 	// ProgressDeadline defines how long workload applier controller will wait for the workload to
 	// reach a successful state in the cluster.
+	// If the workload does not reach a successful state after ProgressDeadline, will stop waiting
+	// and proceed to next rollout.
```

Comment:
it also depend on max failure. Actuall it will stop rolling unless maxFailure is larger than 0

Reply:
have updated the comments